web/rootfs/defaults/meet.conf

```diff
@@ -84,6 +84,17 @@ location = /xmpp-websocket {
 }
 {{ end }}
 
+{{ if $ENABLE_LOAD_TEST }}
```

Comment:
add a | default "0" | toBool or it will fail if you don't specify it.

Reply:
Sorry for the bad code...